filter/slice.go

```diff
@@ -0,0 +1,84 @@
+package filter
+
+import (
+	"fmt"
+
+	"github.com/AdRoll/baker"
+)
+
+var SliceDesc = baker.FilterDesc{
+	Name:   "Slice",
+	New:    NewSlice,
+	Config: &SliceConfig{},
+	Help:   "Slices the source field value using start/end indexes and copies the value to the destination field. If the start index is greater than the field value length, set an empty string to destination",
```

Comment:
We should also document what's happening when the end index is greater than the field length